changehc/delphi_changehc/update_sensor.py

```diff
@@ -162,6 +162,8 @@ def geo_reindex(self, data):
                                                  date_col=Config.DATE_COL)
             # this line should be removed once the fix is implemented for megacounties
             data_frame = data_frame[~((data_frame['county'].str.len() > 5) | (data_frame['county'].str.contains('_')))]
+            # handle rogue \N:
+            data_frame = data_frame[data_frame['county'] != r'\N']
```

Comment:
comment: When I was running this interactively, I recall that at some point the `\N` was appended with extra backslashes (to escape the original backslash, I guess). So the invalid code was coerced into something like `\\\\N`.

suggestion: We should aim to make this filter more robust to potential coercion, since `\` is a special character.

We could do a regex like `".*\N"`, although that might be slow. We could check that the first (and last?) characters of each fips are numeric. Other ways to do this? Maybe check that all fips can be converted into numerics (although this won't work once we start supporting county groups, since those have a format like "55g11").

Reply:
> `\\\\N`

Oh, you ran the indicator with the new code and must not've had a problem with this. Perhaps a python print-formatting thing?

Reply:
So when running change locally, I confirmed that by the time the code reaches geo_reindex, the character was still \N, which was why I have it be literal r'\N'

Reply:
Great, thanks for confirming.